### DISCUSSION TITLE
感觉最近逼站直播亲密度算法又变了

### DISCUSSION THREAD

**Discussion by RtYkk** [andywang425/BLTH#298] [General]
本来不看的直播纯靠脚本每天亲密度基本上都能增加 1000~1100，看的差不多也是这个数。现在变成了看的有时能顶到 1500 的上限，但是不看的纯靠脚本只有 200 了。不知道能解决不？

这个不知道算不算 bug，先在 discussion section 讨论讨论。

**Comment by 1563652588** [comment]
延迟

**Reply by RtYkk** [reply]
什么延迟？反正我最近就是这样的…
![chrome_cnRmftnn5b](https://user-images.githubusercontent.com/45971730/180226070-7b6fe1d2-ddb6-48d1-8f5c-2426594b61ea.png)

**Comment by andywang425** [comment]
B站在[公告](https://link.bilibili.com/p/eden/news#/newsdetail?id=2886)里说了：
> （高峰时段可能有数值加分延迟或不稳定的情况，均属正常现象哦）

**Reply by RtYkk** [reply]
谢了，我再观察一下。

**Reply by RtYkk** [reply]
我感觉是脚本发送观看心跳的部分有点 bug 了，给主播点赞和发弹幕应该没问题，现在看 5 分钟就 100 亲密度，并且公告里没有明确说上限，那就是到 1500 为止。
现在只有我前台看的直播能到 1500，其余都为 0。我猜测啊，脚本在我看一个直播间的时候同时发送我在看其它直播间的心跳。但是现在逼站把这个 ban 了。

**Comment by 66553** [comment]
我也是，连着几天都只有200了
